api/swagger-spec/swagger.yml

```diff
@@ -52,6 +52,26 @@ paths:
           description: unexpected error
           schema:
             $ref: "#/definitions/error"
+  /v3/doctor/{train}/{release}/{uuid}:
```

Comment:
`/v3/doctor/{uuid}` feels appropriate. Can't think of a reason why we'd want to scope by train and release, unless we can't make the API for the payload submission generic enough to accommodate change over time. I'd hate to see a "doctor" API that had to be revved for every release.